zerver/views/registration.py

```diff
@@ -576,7 +575,7 @@ def create_realm(request: HttpRequest, creation_key: Optional[str] = None) -> Ht
 
             try:
                 send_confirm_registration_email(email, activation_url, request.LANGUAGE_CODE)
-            except smtplib.SMTPException as e:
+            except EmailNotDeliveredException as e:
                 logging.error("Error in create_realm: %s", str(e))
```

Comment:
If, as noted above, `EmailNotDeliveredException` doesn't give any useful extra information via `str(e)`, we should take out the `: %s`.